.expeditor/scripts/shared.ps1

```diff
@@ -25,16 +35,15 @@ function Get-Toolchain {
 }
 
 function New-PathString([string]$StartingPath, [string]$Path) {
-  if (-not [string]::IsNullOrEmpty($path)) {
+  if (-not [string]::IsNullOrEmpty($Path)) {
       if (-not [string]::IsNullOrEmpty($StartingPath)) {
-          [string[]]$PathCollection = "$path;$StartingPath" -split ';'
+          [string[]]$PathCollection = "$Path$env:PathSeparator$StartingPath" -split $env:PathSeparator
           $Path = ($PathCollection |
               Select-Object -Unique |
-              Where-Object {-not [string]::IsNullOrEmpty($_.trim())} |
-              Where-Object {test-path "$_"}
```

Comment:
Do we need to check if the path exists? This was throwing off adding `/hab/bin` to the `PATH`.

Reply:
no. we really don't need it